plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/everytrade/EveryTradeBeanV1.java

```diff
@@ -103,7 +102,7 @@ public TransactionCluster toTransactionCluster() {
             buySell,
             related
         );
-        if(equalsToZero(fee)) {
+        if (ZERO.equals(fee)) {
```

Comment:
This is wrong. BigDecimal cannot be compared with `.equals()`  ...you have to do `ZERO.compareTo(fee) == 0` which is happening inside `equalsToZero` method :)